docs/learn/metadata-stores/lancedb.md

```diff
@@ -0,0 +1,17 @@
+# LanceDBMetadataStore
+
+LanceDB keeps one Lance table per feature, appends new metadata versions, and uses the in-memory Polars versioning engine for provenance calculations.
+
+## Installation
+
+The backend relies on [`lancedb`](https://lancedb.com/) which ships with Metaxy's `lancedb` extras.
+
+## Object Stores
+
+Point `uri` at any supported URI (`s3://`, `gs://`, `az://`, `db://`, …) and forward credentials with the platform's native mechanism (environment variables, IAM roles, workload identity, etc.). LanceDB supports local filesystem, S3, GCS, Azure, LanceDB Cloud, and remote HTTP/HTTPS endpoints.
+
+## Storage Layout
```

Comment:
no nested?

Reply:
Compared to i.e. Postgres  (db, scema, table) you would have  db and table where db more or less maps to schema.

So as this all sits on S3 we shoudl clarify if we have one `s3:///some/folder/<<various_feature_dbs>>/some_feature_tables` or if we have `s3:///some/folder/metaxy.lance/some_feature_tables` where `some_feature_tables` refers to Lance internally managed tables

Reply:
TLDR: We could nest on the prefix of the DB but I am not sure if we want that right now